src/exo7 - Collections/exo7.exercise.ts

```diff
@@ -136,13 +167,24 @@ export const odds = new Set([1, 3, 5, 7, 9]);
 //
 // HINT:
 // - Be mindful of the order of operands for the operator you will choose.
+const isInPrimesSet = (entry: number) => {
+ return primes.has(entry)
+}
 
-export const nonPrimeOdds: ReadonlySet<number> = unimplemented();
+export const nonPrimeOdds: ReadonlySet<number> = 
+  pipe (
+    odds,
+    readonlySet.filter(entry => !isInPrimesSet(entry))
+  );
 
 // Construct the set `primeOdds` from the two sets defined above. It should
 // only include the odd numbers that are also prime.
 
-export const primeOdds: ReadonlySet<number> = unimplemented();
+export const primeOdds: ReadonlySet<number> = 
+  pipe (
+    odds,
+    readonlySet.filter(entry => isInPrimesSet(entry))
+  );
```

Comment:
Same here, you can try to use another method from `readonlyArraySet` to code this